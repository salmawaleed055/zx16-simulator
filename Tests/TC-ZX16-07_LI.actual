Loaded 6 bytes into memory
0x0000: F641    addi x1, -5
0x0002: F679    li x1, -5
0x0004: FFC7    Unknown opcode

--- Final State ---
x0: 0x0000
x1: 0xfffb
x2: 0x0000
x3: 0x0000
x4: 0x0000
x5: 0x0000
a0: 0x0000
a1: 0x0000
PC: 0x0004
---------------------

Simulation finished.
