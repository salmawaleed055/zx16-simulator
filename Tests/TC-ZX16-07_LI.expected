0x0000: F679    li x1, 65531
0x0002: FFC7    ecall 0x3FF

--- Final State ---
x0: 0x0000
x1: 0xFFFB
x2: 0x0000
x3: 0x0000
x4: 0x0000
x5: 0x0000
a0: 0x0000
a1: 0x0000
PC: 0x0004
---------------------
Simulation finished.